packages/database/src/modelClasses/Entity.js

```diff
@@ -66,6 +67,7 @@ const ENTITY_TYPES = {
   MEDICAL_AREA,
   NURSING_ZONE,
   FETP_GRADUATE,
+  TRAP,
```

Comment:
Adding new entity type here is not required now. See note: https://github.com/beyondessential/tupaia/blob/867d568f61759c3c524ff039c026f467d6cc6895/packages/database/src/modelClasses/Entity.js#L13

Reply:
Ah thanks for pointing that out @billli0 . Will update the PR. I might keep in the canonical type for this one anyway.